tekton-proposal.md

```diff
@@ -0,0 +1,73 @@
+
+Since the inception of the build-pipeline repo, most active development has happened there (with contributions from Pivotal, CloudBees, Red Hat and others). At this point, Knative Build is largely stable, requiring less than a single full-time engineer effort for requested features, high-priority bug fixes, and scheduled releases. This is most likely a good thing! With the CI/CD features moving to Pipelines, there just aren't many serverless-focused features left for Build to implement at this time, and it could largely be considered "done" until it gets new feature requests from Knative's users and operators.
+
+Pipelines contributors' ambitions continue to expand to address the market for CI/CD for deployment to platforms outside of Kubernetes: raw VMs, mobile devices, IoT devices, and the Kubernetes platform itself. Pipelines may primarily run _on Kubernetes_ but it's not necessarily _for Kubernetes_.
+
+The vaguely overlapping goals and charters of the Build and Pipelines repos continues to cause confusion. Was one going away in favor of the other? Are we maintaining both indefinitely? Are we folding Pipelines into Build? Build into Pipelines? What's even going on. These are not hypothetical questions, these are questions we've heard from users, operators and contributors.
+
+Googlers decided -- without much public input -- ahead of the Knative 0.4 release in February that Pipelines was a separate enough project to warrant its own versioning scheme and release cadence. We also decided (after lengthy trademark review) to christen it "Tekton Pipelines" instead of "Knative Pipelines" at this time.
+
+This was an attempt to signal the Pipelines project's separateness, and help resolve the confusion around Pipelines' relationship with the rest of Knative. Confusion has remained. :)
+
+The above decisions have regrettably all been made and would be difficult (and I would argue even more confusing) to walk back. We can only hope to make better more publicly-informed decisions going forward. This proposal is an attempt to do that. Everything below this paragraph is a **proposal** and is open for discussion. Please leave comments and ask questions.
+
+## Proposal
+
+### Knative Build
+
+[`github.com/knative/build`](https://github.com/knative/build) will be supported as it is for the foreseeable future, overseen by the Knative Build WG which will hold weekly meetings for the foreseeable future. It will continue to do releases co-versioned and timed with the other Knative components.
+
+Its charter and roadmap (which we will write with community input) will focus on enabling better serverless deployment experiences -- source-to-deployment scenarios -- and no more. No workflow, no triggering, just a better developer experience for the Knative serverless platform.
+
+When someone asks, "is Knative's serverless developer experience good?" we should be able to say, "yes, because Knative Build focuses on exactly that."
```

Comment:
This statement is a bit confusing. The DX in Knative probably won't be good because "Build focuses on that" - It's probably the combination of the parts + upcoming work in Client that might drive this good DX.  But I could totally be missing the point you're making here...

Reply:
That's very true, the client team is definitely going to be as critical to a successful DX as Build is, possibly even moreso. The intention of this paragraph was to define what Build is "for" in as few words as possible, not to try to take full responsibility/credit for Knative's developer experience.